src/test/resources/movies-test-schema.graphql

```diff
@@ -7,7 +7,7 @@ type Movie {
   plot: String
   poster: String
   imdbRating: Float
-  genres: [Genre] @relation(name: "IN_GENRE", direction: OUT)
```

Comment:
actually we need to keep the directive
otherwise you don't know the neo4j rel-type.